architecture/additional_concepts/sdn.adoc

```diff
@@ -150,15 +169,19 @@ Finally, if container A connects to an external host, the traffic looks like:
 *_eth0_* _(in A's netns) -> *vethA* -> *br0* -> *tun0* -> (NAT) -> *eth0* (physical device) -> Internet_
 
 Almost all packet delivery decisions are performed with OpenFlow rules in the
-OVS bridge *br0*.  This simplifies the network architecture of the multitenant
-plugin and provides flexible routing and (in the case of the *multitenant*
-plugin) enforceable network isolation.
+OVS bridge *br0*.
+ifdef::openshift-origin[]
+This simplifies the network architecture of the multitenant plug-in and provides
+flexible routing and (in the case of the *multitenant* plug-in) enforceable
+network isolation.
+endif::[]
```

Comment:
Hm... I should have deleted the word "multitenant" from the first half of the sentence. Should be something like:

    Almost all packet delivery decisions are performed with OpenFlow rules in the
    OVS bridge *br0*.  This simplifies the network architecture of the plugin and
    provides flexible routing
    ifdef::openshift-origin[]
    and (in the case of the *multitenant* plug-in) enforceable network isolation
    endif::[]
    .

except that probably won't actually handling the trailing "." correctly